Packs/Slackbot_V2/Integrations/SlackBot/SlackBot.yml

```
@@ -0,0 +1,88 @@
+category: Messaging
+commonfields:
+  id: SlackBot
+  version: -1
+configuration:
+- display: Long running instance
+  name: longRunning
+  required: false
+  type: 8
```

```suggestion
  type: 8
  defaultvalue: 'true'
```

Since the integration relies on long-running, we may want to keep it enabled by default.

---

Done